src/network/networkd-link.c

```diff
@@ -4368,6 +4370,15 @@ int link_save(Link *link) {
                         fputc('\n', f);
                 }
 
+                r = sd_dhcp_lease_get_client_id(link->dhcp_lease, &client_id, &client_id_len);
+                if (r >= 0) {
+                        _cleanup_free_ char *id = NULL;
+
+                        r = sd_dhcp_client_id_to_string(client_id, client_id_len, &id);
+                        if (r >= 0)
+                                fprintf(f, "DHCP4_CLIENTID=%s\n", id);
```

Comment:
I prefer `DHCP4_CLIENT_ID=`. What do you think?

Reply:
Does not matter much as not for user display. But OK.